src/net.js

```diff
@@ -278,7 +278,7 @@ class Network {
        * The URL of the operation.
        * @param {string} rel
        * The type of operation. For example: `preconnect`.
-       * @param {boolean} [crossorigin]
+       * @param {boolean} [cor] crossorigin
```

Comment:
I was under the impression that the name of the argument (cor) came first, and then the description. Perhaps the description should start on a new line instead? If it's "crossorigin" should that be reflected on line 287?